torch/_dynamo/variables/builder.py

```diff
@@ -1729,16 +1758,16 @@ def wrap_to_fake_tensor_and_record(e, tx, *, source: Optional[Source], is_tensor
         )
 
         policy = None
-        if not e.is_nested:
-            # TODO: We should probably support this for nested tensors too
+        is_strided_nt = e.is_nested and e.layout == torch.strided
```

Comment:
curiosity q: Is this a material change? (I didn't know that you can have a nested tensor with potentially multiple layouts)

Reply:
yeah, there's now a "strided layout" nested tensor and a "jagged layout" nested tensor. The former is the old C++ NT, and the latter is a python subclass with the jagged components